src/birthder.js

```diff
@@ -105,6 +105,13 @@
     robot.messageRoom(channelName, message)
   }
 
+  async function isUserActive (robot, user) {
+    // Получаем список всех пользователей вместо обращения к конкретному
```

Comment:
Lets write it in english: `Get list of all users instead of adress one specified user.`